README.md

```diff
@@ -1,33 +1,108 @@
 # dex-operator
 
-This is a operator-sdk based operator to deploy and manage Dex.
+This is a operator-sdk based operator to deploy and manage a Dex server instance.
 
 # Usage
 
-## Using the bundle
+## Option 1: Building your own image and bundle
 
-You'll need operator-sdk 1.9.0+.
+### Requirements
 
-```bash
-git clone https://github.com/identitatem/dex-operator.git
-cd dex-operator
-oc new-project dex-operator
+* You will need operator-sdk 1.9.0+
+* OpenShift cluster
+* Github OAuth Application, for now, we will use github as id provider
+* To narrow the scope, you will want the OCP cluster where you run dex to be using signed certificates, as opposed to the default self-signed certificates. If we use self-signed certificates, of course, we have to distribute the CA to each cluster that try to access the dex service. Of cource, this should be supported in the final cut. You can follow the steps defined here to enable LetsEncrypt certificates: https://github.com/open-cluster-management/sre-tools/wiki/ACM---Day-1#add-an-acme-certificate
 
-# To use prebuilt image. Otherwise, you can reset these variables and build your own.
-export IMAGE_TAG_BASE=quay.io/cdoan/dex-operator
-export VERSION=0.0.2
+### Steps
 
-# this will deploy the operator bundle as a catalog source and deploy the operator from there.
-make sdk-run
+1. build and deploy the operator/controller
 
-# test the DexClient CR
-oc apply -f config/samples/auth_v1alpha1_dexclient.yaml
+    ```bash
+    git clone https://github.com/identitatem/dex-operator.git
+    cd dex-operator
+    oc new-project dex-operator
 
-# test the DexServer CR
-oc apply -f config/samples/auth_v1alpha1_dexserver.yaml
-```
+    # HACK - for now, manually create the clusterrole/binding to give dex access to dex resources
+    oc apply -f hack/community/rbac-dex-operator-dexsso-community.yaml
+
+    # Set the path to your quay repo.
```

Comment:
Not necessary for this PR but we should note here that the repo needs to be public or a pull secret needs to be configured.

Reply:
Yep, that makes sense,